pages/base/header_start.inc

```diff
@@ -8,7 +8,8 @@
 
 <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.11.2/css/all.css">
 <script defer src="https://use.fontawesome.com/releases/v5.11.2/js/all.js"></script>
-<script src="https://unpkg.com/vue"></script>
+<script src="https://unpkg.com/vue@3.3.8/dist/vue.global.js"></script>
```

Comment:
Eventually, it would be nice to conditionally include scripts depending on whether we're developing or in production. We can include the base files for us during development, and minified files that have been optimized to load as quickly as possible in production environments.